main.go

```diff
@@ -22,6 +22,8 @@ import (
 	"os"
 
 	mgmt "github.com/purpleidea/mgmt/lib"
+
+	_ "github.com/purpleidea/mgmt/resources/http"
```

Comment:
I think this should go in resources.go instead. (So that people importing `resources` get it too.) If you could add a comment like: `// resource registration` or something please.